jooby/src/main/java/org/jooby/Asset.java

```diff
@@ -324,7 +326,13 @@ default String etag() {
     b.append("W/\"");
 
     BaseEncoding b64 = BaseEncoding.base64();
-    int lhash = resource().hashCode();
+    int lhash = 0;
+    try {
+      lhash = resource().toURI().hashCode();
+    } catch (URISyntaxException e) {
+      throwIfUnchecked(e);
+      throw new RuntimeException(e);
```

Comment:
URISyntaxException isn't an unchecked exception and I don't like to wrap exception as runtime. It is just noise